packages/astro/test/astro-slots.test.js

```diff
@@ -1,5 +1,3 @@
-/**
- * UNCOMMENT: add Astro slot support
 import { expect } from 'chai';
 import cheerio from 'cheerio';
```

Comment:
This is not in the scope of this PR at all but I would love to drop `cheerio` for `linkedom`.